scripts/run_changed.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
```

Comment:
Nit: 2020

Reply:
Thanks, also fixed this in the template that auto-adds the license headers.